codalab/bundles/uploaded_bundle.py

```diff
@@ -26,8 +26,9 @@ class UploadedBundle(NamedBundle):
     )
     METADATA_SPECS.append(
         MetadataSpec('link_format', str, 'Link format of bundle. Can be equal to'
-            '"raw" or "zip" (only "raw" is supported as of now).', optional=True)
+                                         '"raw" or "zip" (only "raw" is supported as of now).', optional=True)
```

Comment:
Nit: Is this spacing too much? Maybe one less tab?

Reply:
oh, this indentation is suggested by pycharm, to be at the same place of the beginning of former part.